rover/src/main/kotlin/io/rover/rover/ui/LayoutableViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package io.rover.rover.ui
+
+import android.graphics.Rect
+
+/**
+ * Any View Model that implements this interface can be laid out in in the vertically
+ * scrollable plane of a [ScreenViewModel].
+ */
+interface LayoutableViewModel {
+    // TODO: in here go parameters common to the display of both Rows and Blocks.  So, possibly,
```

Comment:
Remind me later to discuss button states, specifically how they apply to borders and backgrounds.